.github/workflows/node.js.yml

```diff
@@ -91,9 +91,6 @@ jobs:
     - name: Build app
       run: npm run build --if-present
 
-    - name: Build Elm static Db
-      run: npm run db:build
```

Comment:
Note to reviewers: this is not necessary as this is called by the `npm run build` call above.